hack/make-rules/verify.sh

```diff
@@ -147,10 +147,11 @@ function run-checks {
   local -r runner=$2
 
   local t
-  for t in $(ls ${pattern})
+  for t in ${pattern}
   do
-    local check_name="$(basename "${t}")"
-    if [[ ! -z ${WHAT:-} ]]; then
+    local check_name
+    check_name="$(basename "${t}")"
```

Comment:
```
In ./hack/make-rules/verify.sh line 151:
    local check_name="$(basename "${t}")"
          ^--------^ SC2155: Declare and assign separately to avoid masking return values.
```